### ISSUE TITLE
Off Command Missing

### ISSUE THREAD

**Issue by macjeff** [simplextech/udi-poly-linktap#1]
The API said Instant Off is an option.

I want to write program in ISY where THEN is on and ELSE is off.  

I tried Instant on for Zero Minutes but that wont work.

Could you add the OFF command?

**Comment by simplextech** [comment]
The function should handle the duration 0 being off.  Please post logs so I can see what is happening.  I don't have a unit installed currently (winter, raining) so it's going to take time to test.

**Comment by macjeff** [comment]
sorry I missed this.  Will get logs for you

**Comment by macjeff** [comment]
I just got the valves and really just testing them now.  So I have 3 of them linked but only 1 is on.
I started from scratch.  Deleted the node server and uninstalled it.  Reinstalled it.   Rebooted admin console server as it asked.  Created a new API key and entered it and user name.  All my node names showed up. 

Did instant on 1 minute.  It turned on fine BUT the status showed all zeros for watering time remaining, total, time elapsed.   I then changed to instant on zero and it would not turn off.  After a minute it turned off on its own.  I tried a few times and ONCE I got the status to show up but at no time could I turn off.  I see a lot of errors in the logs so something is wrong. 

I did hit query and update profile somewhere in there so if you see it rediscover that was my fault.

 Logs attached

[LinkTap_logs_2020-2-19_230954.zip](https://github.com/simplextech/udi-poly-linktap/files/4228741/LinkTap_logs_2020-2-19_230954.zip)

**Comment by simplextech** [comment]
Looking at the logs it's showing a bad request to the API.  Couple of questions and then I'll have to do some digging

1. Version of Polyglot
2. Looks like local but is this local or cloud?
3. rPi installed or on Polisy?
4. LinkTap device versions?  New one's with water meter or older?

**Comment by macjeff** [comment]
Good news is I live in VA so water wont even be turned on until April 1 or so.  I have learned not to wait until last minute on this stuff.   So if you want to PM me I can give you access to the system since none of the valves are really hooked up to water now anyways

1. On polisy it says Version 2.2.9-5   Is that what you need?
2. Polisy Local
3. Polisy
4. New ones with water meter. Just purchased.

**Comment by macjeff** [comment]
Did you get a chance to look at my reply?  Anything else you need from me?

**Comment by macjeff** [comment]
After a day or two I got this (no exact time on the log).  Everything was blank in the ISY.  I restarted the node server and did a query and its back working except still the Instant On ZERO for off wont work.

requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.link-tap.com', port=443): Max retries exceeded with url: /api/getAllDevices (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x8035cbdd0>: Failed to establish a new connection: [Errno 8] Name does not resolve'))
